<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoordinateSystemAdjustmentWidget</class>
 <widget class="QDialog" name="CoordinateSystemAdjustmentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coordinate System Specification</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Data Set:</string>
        </property>
        <property name="buddy">
         <cstring>dataSetNameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="dataSetNameEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Data Set &amp;Type:</string>
        </property>
        <property name="buddy">
         <cstring>dataSetTypeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="dataSetTypeEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Coordinate Value &amp;Ranges:</string>
        </property>
        <property name="buddy">
         <cstring>coordinateValueRangesEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPlainTextEdit" name="coordinateValueRangesEdit">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>X:
Y:
Center:
Relative Position of (0; 0):</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Coordinate System Type:</string>
        </property>
        <property name="buddy">
         <cstring>coordinateSystemTypeCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="coordinateSystemTypeCombo">
        <item>
         <property name="text">
          <string>(unspecified)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Geographic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Metric (global)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Metric (local)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&amp;Geographic Coordinate System:</string>
        </property>
        <property name="buddy">
         <cstring>geoCoordSystemEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="geoCoordSystemEdit">
        <property name="text">
         <string>WGS 84</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&amp;Metric Coordinate System:</string>
        </property>
        <property name="buddy">
         <cstring>metricCoordSystemEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="metricCoordSystemEdit">
        <property name="text">
         <string>UTM</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>&amp;Unit of Measurement:</string>
        </property>
        <property name="buddy">
         <cstring>unitEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Reference &amp;Point</string>
        </property>
        <property name="buddy">
         <cstring>autoSetReferencePointButton</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="autoSetReferencePointButton">
        <property name="text">
         <string>Set Reference Point To...</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QCheckBox" name="referencePointGlobalCheckBox">
        <property name="text">
         <string>Gl&amp;obal (geographic)</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QWidget" name="refPointGlobalWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Latitude:</string>
           </property>
           <property name="buddy">
            <cstring>refGlobalLatitudeSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="refGlobalLatitudeSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-90.000000000000000</double>
           </property>
           <property name="maximum">
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Longitude:</string>
           </property>
           <property name="buddy">
            <cstring>refGlobalLongitudeSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="refGlobalLongitudeSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-180.000000000000000</double>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QCheckBox" name="referencePointLocalCheckBox">
        <property name="toolTip">
         <string>(0; 0) is the northwestern corner, (0.5; 0.5) is the center of the data set.</string>
        </property>
        <property name="text">
         <string>&amp;Local (relative)</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QWidget" name="refPointLocalWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Northing:</string>
           </property>
           <property name="buddy">
            <cstring>refLocalRelativeNorthingSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="refLocalRelativeNorthingSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Easting:</string>
           </property>
           <property name="buddy">
            <cstring>refLocalRelativeEastingSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="refLocalRelativeEastingSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="unitEdit">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dataSetNameEdit</tabstop>
  <tabstop>dataSetTypeEdit</tabstop>
  <tabstop>coordinateValueRangesEdit</tabstop>
  <tabstop>coordinateSystemTypeCombo</tabstop>
  <tabstop>geoCoordSystemEdit</tabstop>
  <tabstop>metricCoordSystemEdit</tabstop>
  <tabstop>unitEdit</tabstop>
  <tabstop>autoSetReferencePointButton</tabstop>
  <tabstop>referencePointGlobalCheckBox</tabstop>
  <tabstop>refGlobalLatitudeSpinBox</tabstop>
  <tabstop>refGlobalLongitudeSpinBox</tabstop>
  <tabstop>referencePointLocalCheckBox</tabstop>
  <tabstop>refLocalRelativeNorthingSpinBox</tabstop>
  <tabstop>refLocalRelativeEastingSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
